### ISSUE TITLE
Setting targetSdkVersion > 23 causes FullPath errors for animated selectors

### ISSUE THREAD

**Issue by codeversed** [nickbutcher/plaid#132]
If you set targetSdkVersion to 24 or 25, you get some weird vector errors like below. It seems to be affecting animated selectors and transition data. It does not just error for alpha but seems to error on any transition property. 

```
java.lang.IllegalArgumentException: Property: alpha is not supported for FullPath
   at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.createRTAnimatorForFullPath(AnimatedVectorDrawable.java:1325)
   at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.createRTAnimator(AnimatedVectorDrawable.java:1248)
   at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.parseAnimatorSet(AnimatedVectorDrawable.java:1220)
   at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.init(AnimatedVectorDrawable.java:1187)
   at android.graphics.drawable.AnimatedVectorDrawable.ensureAnimatorSet(AnimatedVectorDrawable.java:851)
   at android.graphics.drawable.AnimatedVectorDrawable.start(AnimatedVectorDrawable.java:834)
   at android.graphics.drawable.AnimatedStateListDrawable$AnimatedVectorDrawableTransition.start(AnimatedStateListDrawable.java:339)
   at android.graphics.drawable.AnimatedStateListDrawable.selectTransition(AnimatedStateListDrawable.java:226)
   at android.graphics.drawable.AnimatedStateListDrawable.onStateChange(AnimatedStateListDrawable.java:146)
   at android.graphics.drawable.Drawable.setState(Drawable.java:735)
   at android.widget.ImageView.drawableStateChanged(ImageView.java:1246)
   at android.view.View.refreshDrawableState(View.java:18045)
```

**Comment by dgngulcan** [comment]
Have you found a way to fix this? thanks

**Comment by nickbutcher** [comment]
@dgngulcan haven't had a chance to look into this yet… will report back here when I do!

**Comment by alexjlockwood** [comment]
Hey Nick, I noticed that the 'search to back' animated vector drawable crashes due to a similar error as well:

    java.lang.IllegalArgumentException: Property: translateX is not supported for FullPath

It looks like before Android N MR1, AnimatedVectorDrawable would simply ignore animators that weren't well formed. In Android N MR1, the framework team decided to start crashing these apps instead (but only apps that target Android N MR1 or greater: https://github.com/android/platform_frameworks_base/blob/nougat-dr1-release/graphics/java/android/graphics/drawable/AnimatedVectorDrawable.java#L224).

I'm not sure which animation the OP of this bug was referring to, but the reason why the 'search to back' animation was crashing was because there is an animator in res/animator/searchback_show_arrow_head.xml that attempts to animate the 'translateX' property of a path (which is not supported... apparently you can only translate a group containing one or more paths). In order to get it working again, I had to create an 'arrow_head_group' and animate/translate that in the AnimatedVectorDrawables instead.

Another option would be to just remove the translation animation... if you haven't noticed that this hasn't been working up until now, then maybe it isn't even necessary to have in the first place. :)

Anyway, TL;DR... these crashes mean you're attempting to animate a property incorrectly... I'll let you know if I encounter any others (I've only seen it in search to back so far).

**Comment by alexjlockwood** [comment]
BTW, I also had to change the translationX values as well... they are currently animated between '100' and '20'... but this was way too much. Using '10' and '2' seemed like they were closer to what you would want... maybe a typo?

**Comment by nickbutcher** [comment]
@alexjlockwood thanks so much for the investigation! Fix incoming.
